src/server/getPageTemplate.js

```diff
@@ -21,11 +21,14 @@ module.exports = ({ head = [""], path = "/" }) => {
   return `<!DOCTYPE html>
 <html lang="en">
   <head>
-    ${head.join("\n")}
-    ${getRemoteEntryScript(path)}
+    <script>
+      window.config=${JSON.stringify(config)}
+    </script>
   </head>
```

Comment:
This will conflict with my dev-server PR, where scripts like these are brought in via the `server.ts` file, FYI

Reply:
We can federate the module instead of doing this. This is to prove this works.